google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Mutation.java

```diff
@@ -62,6 +62,13 @@ public static Mutation create() {
     return new Mutation(false);
   }
 
+  /** Wraps the List of protobuf {@link com.google.bigtable.v2.Mutation}. */
+  public static Mutation fromProto(List<com.google.bigtable.v2.Mutation> protos) {
+    Mutation mutation = new Mutation(false);
```

Comment:
i'm concerned about this assuming that all of the mutations contain serverside timestamps

Reply:
What would you suggest?  Should we add a boolean, and maybe a check for server-side timestamps?

Reply:
(`toProto` is required for migrating cloud-bigtable-client's BulkMutation infrastructure)

Reply:
I think easiest would be call this `fromProtoUnsafe` and mark is as `InternalOnly` with similar warning as `createUnsafe`? Also, can you move the method to after createUnsafe()?